packages/connect/src/Detector.test.ts

```diff
@@ -0,0 +1,8 @@
+import * as smoldot from 'smoldot';
+
+describe('Test Detector (Fake)', () => {
+    it('works', () => {
+        console.log(smoldot);
```

Comment:
Remove the `cosnole.log`  and can we make this test do something a little more sensible at least?  Maybe check that `Detector.connect` returns something and doesn't error that's a reasonable happy path case

Reply:
This test was in the scope of "fixing jest to work with substrate connect".
I will remove it for now. Tests will be implemented on different PR as this one is already too large.